OsmAnd/res/values/strings.xml

```diff
@@ -58,37 +58,37 @@
     <string name="weather_wind_kilimeters_per_hour">Kilometers per hour</string>
     <string name="weather_wind_meters_per_second">Meters/second</string>
     <string name="weather_widgets_secondary_desc">Measurement units can be changed in the Weather plugin settings.</string>
-    <string name="air_pressure_widget_desc">Displays the air pressure for the current map center.</string>
-    <string name="clouds_widget_desc">Shows the cloud coverage for the current map center.</string>
-    <string name="wind_widget_desc">Displays the wind speed for the current map center.</string>
-    <string name="precipitation_widget_desc">Shows the precipitation for the current map center.</string>
-    <string name="temperature_widget_desc">Shows the temperature for the current map center.</string>
-    <string name="weather_widget_group_desc">A set of widgets showing various weather information.</string>
+    <string name="air_pressure_widget_desc">Shows air pressure for current map center.</string>
+    <string name="clouds_widget_desc">Shows clouds for current map center.</string>
```

Comment:
In meteorologic wording cloud coverage is correct.

Reply:
I'll change it back, but this is my thinking:
I think "cloud coverage" can be expressed in percentage, or as a median.
More-so in other languages "cloud coverage" is correct here.
"Cloud cover (also known as cloudiness, cloudage, or cloud amount)" is the other meaning, as a distribution for different places. "Clouds" wasn't very good, so maybe "cloudiness".